2020-03-02 17:01:24,734 INFO [root] [__init__.py(<module>:15)] - 测试日志是否能够正常打印2！
2020-03-02 17:01:24,837 INFO [root] [test_login.py(test01_login_success:20)] - 登陆成功的结果为：{'success': True, 'code': 10000, 'message': '操作成功！', 'data': '5a64e696-5e1b-48e6-83c5-dd836948a1cc'}
2020-03-02 17:01:24,921 INFO [root] [test_login.py(test02_username_is_not_exist:29)] - 测试用户名不存在的结果为：{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2020-03-02 17:01:25,006 INFO [root] [test_login.py(test03_password_is_error:38)] - 测试密码错误的结果为：{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2020-03-02 17:01:25,091 INFO [root] [test_login.py(test04_none_params:47)] - 无参的结果为：{'success': False, 'code': 99999, 'message': '抱歉，系统繁忙，请稍后重试！', 'data': None}
2020-03-02 17:01:25,172 INFO [root] [test_login.py(test05_username_is_null:55)] - 用户名为空的结果为：{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2020-03-02 17:01:25,258 INFO [root] [test_login.py(test06_password_is_empty:63)] - 密码为空的结果为：{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2020-03-02 17:01:25,339 INFO [root] [test_login.py(test07_less_mobile:72)] - 缺少mobile结果为：{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2020-03-02 17:01:25,428 INFO [root] [test_login.py(test08_less_password:81)] - 缺少password结果为：{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2020-03-02 17:01:25,512 INFO [root] [test_login.py(test09_add_params:92)] - 增加1个参数结果为：{'success': True, 'code': 10000, 'message': '操作成功！', 'data': '19ecf772-6bab-4983-9b9e-614b72472ec1'}
2020-03-02 17:01:25,590 INFO [root] [test_login.py(test10_error_params:102)] - 错误参数结果为：{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
